rapids_build_backend/impls.py

```diff
@@ -234,11 +264,23 @@ def get_requires_for_build_wheel(config_settings):
             backend := _get_backend(config.build_backend),
             "get_requires_for_build_wheel",
         ):
-            requires.extend(
-                backend.get_requires_for_build_wheel(
-                    _remove_rapidsai_from_config(config_settings)
+            if config.build_backend == "setuptools.build_meta":
+                _check_setup_py(setup_py_contents=utils._get_setup_py())
+                # prior to https://github.com/pypa/setuptools/pull/4369 (May 2024),
```

Comment:
Yeah it does look like it made it into `setuptools==70.0`, so that's possible.

https://github.com/pypa/setuptools/compare/v70.0.0

I chose not to do that type of comparison in the interest of keeping this simpler. I thought the risk of "sometimes an unnecessary build dependency on `wheel` gets pulled in" was worth it in exchange for keeping this simpler.

But I'm indifferent. If you want to add that version condition as a guard on this I'd be happy to.

Reply:
Yeah I think keeping it simple is fine. No change needed.